app/views/components/gds/link.scala.html

```diff
@@ -0,0 +1,19 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@(messageKey: String = "", href: Call)(implicit messages: Messages)
```

Comment:
I would go towards passing the message itself. Then you don't have to provide `Messages` too.

Reply:
You are right.  That is the new/preferred way